api/Infrastructure/RiderParticipation/TeamselectionClient.cs

```diff
@@ -0,0 +1,44 @@
+using SpoRE.Infrastructure.Base;
+
+namespace SpoRE.Infrastructure.Database.Teamselection;
+
+public class TeamselectionClient
+{
+    SqlDatabaseAdapter DatabaseClient;
+    public TeamselectionClient(SqlDatabaseAdapter databaseClient)
+    {
+        DatabaseClient = databaseClient;
+    }
+
+    public async Task<Result<List<RiderParticipationRider>>> Get(int raceId)
+        => await DatabaseClient.Get<RiderParticipationRider>(
+            @$"SELECT * FROM rider_participation 
+            INNER JOIN rider USING(rider_id)
+            WHERE race_id = @raceId",
+            new List<QueryParameter>(){new(raceId)}
+        );
+}
+
+public record RiderParticipationRider // TODO move naar eigen file/folder, dit is effectief ook een FE model, Kunnen we die TS models laten genereren?
```

Comment:
De modellen in de DB beginnen met kleine letters. Dus dit model moet je aanpassen. Daarom kreeg je waarschijnlijk allemaal lege data. Ik heb dat nog niet gedaan in mijn stage ding. In gevallen van simpel data ophalen kunnen we waarschijnlijk beter gewoon modellen definieren zonder hoofdletters. iig voor nu.
En omdat jij 0 logica hebt in je Service kan die eigenlijk ook verwijderd worden.

Reply:
![image](https://user-images.githubusercontent.com/15751288/218479565-d977b8b9-b68d-43dd-abb5-bfe7bde610b7.png)
We kunnen het ook op deze manier doen